src/System.Collections.Specialized/src/System.Collections.Specialized.builds

```diff
@@ -4,8 +4,14 @@
   <ItemGroup>
     <Project Include="System.Collections.Specialized.csproj" />
     <Project Include="System.Collections.Specialized.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
+    </Project>
+    <Project Include="System.Collections.Specialized.csproj">
       <TargetGroup>net46</TargetGroup>
```

Comment:
I think you will also need the net463 entry along with the default empty ns17 entry. 

I would prefer we do this as part of this PR as it is introducing a bunch more logic and forked code which isn't the case for most of the other existing projects. While most of them have some depproj's I think most of them don't have much if any #ifdef'ed code currently.